app/models/query/sums.rb

```diff
@@ -76,7 +76,7 @@ def total_sum_of(column)
   end
 
   def sum_of(column, collection)
-    return unless should_be_summed_up?(column)
+    return nil unless should_be_summed_up?(column)
```

Comment:
They are... I guess I would usually not have changed it (because I touched nothing else in that method).
But my reasoning is something like: If a method usually returns a non-nil value, be explicit about it returning nil.